### PR TITLE
Feature/four 11726: FOUR-11509: Create the new component to load the submenus in right

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
image-20231107-192312

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11726

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-47ba7ce1f1.engk8s.processmaker.net

File: resources/js/admin/settings/components/SettingsMenu.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+<script>
+import SettingsListing from './SettingsListing';
+import SettingsMenuCollapse from './SettingsMenuCollapse';
+
+export default {
+  components: { SettingsListing, SettingsMenuCollapse },
+  data() {
+    return {
+      currentTab: 0,
+      group: "Action by Email",
```
Comment (pmPaulis) [inline resources/js/admin/settings/components/SettingsMenu.vue:33]: Please not use a harcoded value, what happen in the Action by Email label was renamed?
Maybe the current value selected is the firstOption from the firstMenu

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by pmPaulis** [comment]
@luNunezProcessmaker 
This is not working we can see the following issues:
image

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-47ba7ce1f1.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-47ba7ce1f1.engk8s.processmaker.net

**Pull request merged**